src/SFA.DAS.ApprenticeCommitments.Api/Controllers/ConfirmHowApprenticeshipWillBeDeliveredController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.ApprenticeCommitments.Application.Commands.HowApprenticeshipWillBeDeliveredCommand;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticeCommitments.Api.Controllers
+{
+    public class ConfirmHowApprenticeshipWillBeDeliveredRequest
+    {
+        public bool HowApprenticeshipDeliveredCorrect { get; set; }
+    }
+
+    [ApiController]
+    public class ConfirmHowApprenticeshipWillBeDeliveredController : ControllerBase
+    {
+        private readonly IMediator _mediator;
+
+        public ConfirmHowApprenticeshipWillBeDeliveredController(IMediator mediator) => _mediator = mediator;
+
+        [HttpPost("apprentices/{apprenticeId}/apprenticeships/{apprenticeshipId}/howapprenticeshipwillbedeliveredconfirmation")]
+        [HttpPost("apprentices/{apprenticeId}/apprenticeships/{apprenticeshipId}/{commitmentStatementId}/HowApprenticeshipWillBeDeliveredConfirmation")]
+        public async Task HowApprenticeshipWillBeDelivered(
+            Guid apprenticeId, long apprenticeshipId, long commitmentStatementId,
+            [FromBody] ConfirmHowApprenticeshipWillBeDeliveredRequest request)
+        {
+            await _mediator.Send(new HowApprenticeshipWillBeDeliveredCommand(
+                (apprenticeId, apprenticeshipId, commitmentStatementId),
```

Comment:
I really don't like the Id class here. But passing it in in this way, adds confusion

Reply:
I got that impression from all the comments :)  It was an attempt to group the IDs into one - a commitment statement is for an apprenticeship for an apprentice.  I had thought it would clarify that all three IDs are needed to be clear about what's being updated.  It seems that it may just be adding confusion though.  I'll have another look.

Reply:
I reworked the confirmations.  Now there is one `ConfirmCommand` that handles confirming every aspect of the apprenticeship.  This removed all the duplication of the apprentice/apprenticeship/commitment-statement IDs that I was originally trying to solve with the `ApprenticeshipCommitmentStatementId` amalgamation.  Therefore, it has also been removed.